app/views/claim/view-claim.html

```diff
@@ -69,12 +69,11 @@ <h1 class="table-heading">Visit</h1>
 
     {% include "partials/view-claim/visit.html" %}
 
-    <div id='claim-expense'></div>
     <div id='approve-cost'></div>
     <h1 class="table-heading">Claim</h1>
-
+    <div id='claim-expense'>
     {% include "partials/view-claim/claim.html" %}
-
+    </div>
```

Comment:
unnecessary tag, the styles should be against `partials/view-claim/claim.html` table as class, so it is obvious which table td it applies too